generated/niscope/session.py

```diff
@@ -2718,11 +2715,11 @@ def fetch_into(self, waveform, relative_to=enums.FetchRelativeTo.PRETRIGGER, off
 
             num_records (int): Number of records to fetch. Use -1 to fetch all configured records.
 
-            timeout (float): The time to wait in seconds for data to be acquired; using 0 for this parameter tells NI-SCOPE to fetch whatever is currently available. Using -1 for this parameter implies infinite timeout.
+            timeout (float in seconds or datetime.timedelta): The time to wait in seconds for data to be acquired; using 0 for this parameter tells NI-SCOPE to fetch whatever is currently available. Using -1 for this parameter implies infinite timeout.
 
 
         Returns:
-            wfm_info (list of WaveformInfo): Returns an array of classed with the following timing and scaling information about each waveform:
+            wfm_info (WaveformInfo): Returns an array of classed with the following timing and scaling information about each waveform:
```

Comment:
Ignoring the odd wording which came from metadata (`array of classed`??), the type no longer says this is a list.
Is this a codegen bug introduced?

I'm ok fixing this in a separate PR.